app/src/main/java/com/sun/moviedb_48/data/source/remote/fetchdata/ParseUrlToJson.java

```diff
@@ -67,4 +69,39 @@ public Movie parseMovieFromJson(JSONObject jsonObjectMovie) {
         }
         return movie;
     }
+
+    public ArrayList<Person> getListPersonFrom(String jsonString, @PersonType String personType) {
+        ArrayList<Person> people = new ArrayList<>();
+        try {
+            JSONObject jsonObject = new JSONObject(jsonString);
+            if (personType.equals(PersonType.CREW)) {
+                JSONArray jsonArray = jsonObject.getJSONArray(Person.PersonEntry.CREW);
+                if (jsonArray.length() != 0) {
+                    for (int i = 0; i < Person.PersonEntry.NUMBER_OF_CREW; i++) {
```

Comment:
tại sao lại biết trước được số lượng Crew vậy, nếu database thay đổi thì số Crew này không còn chính xác, tại sao lại không dùng jsonArray.leng() nhỉ?

Reply:
Em chỉ lấy 1 crew đầu tiên cũng chính là director luôn ạ

Reply:
Nếu muốn lấy 1 giá trị thì k cần dùng for đâu.

Reply:
> Nếu muốn lấy 1 giá trị thì k cần dùng for đâu.

Okay anh